udata/core/dataset/api_fields.py

```diff
@@ -101,6 +101,14 @@
 community_resource_page_fields = api.model(
     'CommunityResourcePage', fields.pager(community_resource_fields))
 
+#: Default mask to make it lightweight by default
+DEFAULT_MASK = ','.join((
```

Comment:
In this PR, the position was:
- mask as string by default
- as array only when needed or better (here only to be able to have a line under 80 char)

On the client side I wanted to avoir array joining every time. But using joined array by default is fine too me too.